### PR TITLE
Replaced single quotes with double quotes line 29 : az aks list --que…

### PR DESCRIPTION
**Pull request by virajk009**
…ry "[].

az aks list --query "[].{resourceGroup:resourceGroup, AksName:name, clientId: podIdentityProfile.userAssignedIdentities[0].identity.clientId}" -o table

### CODE REVIEW

File: docs/2.podidentity.md [hunk 1]
```diff
@@ -26,7 +26,7 @@ HIGHLY RECOMMENDED: Verify pod identity is operational on each AKS cluster.
 1. Obtain Subscription ID, AKS Resource Group name, AKS name, and Client ID of each pod identity:
 ```bash
 az account show --query "{subscriptionId: id}" -o table
-az aks list --query '[].{resourceGroup:resourceGroup, AksName:name, clientId: podIdentityProfile.userAssignedIdentities[0].identity.clientId}' -o table
+
```
Comment (rjygraham) [inline docs/2.podidentity.md:29]: Looks like the incoming change actually removed the line. Can you add it back in and then I'll approve/merge the PR.

File: docs/2.podidentity.md [hunk 2]
```diff
@@ -26,7 +26,7 @@ HIGHLY RECOMMENDED: Verify pod identity is operational on each AKS cluster.
 1. Obtain Subscription ID, AKS Resource Group name, AKS name, and Client ID of each pod identity:
 ```bash
 az account show --query "{subscriptionId: id}" -o table
-az aks list --query '[].{resourceGroup:resourceGroup, AksName:name, clientId: podIdentityProfile.userAssignedIdentities[0].identity.clientId}' -o table
```
Comment (virajk009) [inline docs/2.podidentity.md:29]: az aks list --query "[].{resourceGroup:resourceGroup, AksName:name, clientId: podIdentityProfile.userAssignedIdentities[0].identity.clientId}" -o table

**Review by virajk009 — COMMENTED** [review]
az aks list --query "[].{resourceGroup:resourceGroup, AksName:name, clientId: podIdentityProfile.userAssignedIdentities[0].identity.clientId}" -o table

**Comment by virajk009** [comment]
Done.

**Pull request closed without merging**